detection_rules/eswrap.py

```diff
@@ -236,3 +241,182 @@ def collect_events(ctx, agent_hostname, index, agent_type, rta_name, rule_id, vi
     except AssertionError as e:
         error_msg = 'No events collected! Verify events are streaming and that the agent-hostname is correct'
         client_error(error_msg, e, ctx=ctx)
+
+
+@es_group.group('beta')
+def es_beta():
+    """BETA helper commands for integrating with Elasticsearch."""
```

Comment:
maybe change all the `beta` to `experimental`?

Reply:
all around, I think `beta` should be replaced with `experimental`. I think it's more consistent with how we use the language in other parts of the stack. And to bring more attention to it here:
```suggestion
@es_group.group('experimental')
def es_experimental():
    """[Experimental] Helper commands for integrating with Elasticsearch."""
```